client/src/partials/development/reports/core.html

```diff
@@ -0,0 +1,73 @@
+<header data-header>
+  {{ReportCtrl.title}}
+</header>
+
+<!-- TODO: If page structure should be replaced with implicit correctness through -->
+<!-- dynamic, database driven routes -->
+<nav ng-if="::ReportCtrl.state.isValid">
+  <button ng-click="ReportCtrl.hookGeneration()" class="btn btn-primary btn-default">Create new {{ReportCtrl.title}} Document</button>
+
+  <div class="pull-right">
+    <button disabled class="btn btn-default">Manage recurring report</button>
+    <button disabled class="btn btn-default">Settings</button>
+  </div>
+</nav>
+
+<main>
+  
+  <!-- Binding one way declutters the digest loop after the initial page load -->
+  <section ng-if="::ReportCtrl.state.isInvalid">
+    <div class="alert alert-warning">
+      Report key '{{ReportCtrl.reportKey}}' is not supported.
+    </div>
+  </section>
+  
+  <section ng-if="::ReportCtrl.state.isValid">
+  
+    <!-- TODO: Review how expensive the conditional templating below is -->
+    <!-- Searching for archives -->
+    <div ng-if="ReportCtrl.state.isLoading" ng- class="panel panel-default">
+      <div class="panel-body">
+        <i style="padding-right : 0.5em;" class="glyphicon glyphicon-search"></i> <strong>Searching archives...</strong>
+      </div>
+    </div>
+
+    <!-- No archive data found -->
+    <div ng-if="ReportCtrl.archivesEmpty" class="alert alert-info">
+      <i style="padding-right : 0.5em;" class="glyphicon glyphicon-folder-open"></i> <strong> No reports found in the archive.</strong> Use the 'Create new {{ReportCtrl.title}} Document' button to create a new report.
+    </div>
+
+    <!-- Found archives -->
+    <div ng-if="ReportCtrl.archivesEmpty===false && ReportCtrl.state.suppressSuccessDisplay===false" class="alert alert-success alert-dismissible">
```

Comment:
Don't we have an `<alert>` directive available to us with ui-bootstrap?  If not, we should get it to remove boilerplate code.